spec/shared_context.rb

```diff
@@ -0,0 +1,53 @@
+require 'vcr'
+
+class KubevirtVCR
+	attr_reader :service,
+	            :host,
+	            :port,
+	            :token
+
+  def initialize(options)
+    @vcr_directory = options[:vcr_directory]
+    @service_class = options[:service_class]
+
+    use_recorded = !ENV.key?('KUBE_HOST') || ENV['USE_VCR'] == 'true'
+    if use_recorded
+      Fog.interval = 0
+      @host = "192.168.122.51"
+      @port = "8443"
+    else
+      @host = ENV['KUBE_HOST']
+      @port = ENV['KUBE_PORT']
+    end
+
+    VCR.configure do |config|
+      config.allow_http_connections_when_no_cassette = true
+      config.hook_into :webmock
+
+      if use_recorded
+        config.cassette_library_dir = ENV['SPEC_PATH'] || @vcr_directory
+        config.default_cassette_options = {:record => :none}
+        config.default_cassette_options.merge! :match_requests_on => %i[method uri body]
+      else
+        config.cassette_library_dir = "spec/debug"
```

Comment:
if were using this module for recording as well, and expecting the output to be under `spec/debug` - please add it to project's .gitignore

Reply:
OK